bin/yebash

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+LD_PRELOAD=/usr/local/lib/libyebash.so /bin/bash
```

Comment:
I wouldn't hardcode the path to `/usr/local`, because it depends on `CMAKE_INSTALL_PREFIX`. Instead, I suggest to use [configure_file](https://cmake.org/cmake/help/v3.0/command/configure_file.html).